py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -70,12 +247,31 @@ def addmm_replacement(
     )
 
 
-@register_decomposition(torch.ops.aten.reciprocal.default, registry=DECOMPOSITIONS)
+@register_decomposition(
+    torch.ops.aten.reciprocal.default, registry=CUSTOM_DECOMPOSITIONS
+)
 def reciprocal_replacement(
     input_: torch.Tensor,
 ) -> torch.Tensor:
     return torch.div(1, input_)
 
 
-def get_decompositions():
-    return DECOMPOSITIONS
+def get_decompositions(
```

Comment:
The dictionary returned by `get_decompositions` is either `TORCH_DECOMPOSITIONS` or `TORCH_EXPERIMENTAL_DECOMPOSITIONS` concatenated with our `CUSTOM_DECOMPOSITIONS`